pandapower/build_bus.py

```diff
@@ -329,9 +329,23 @@ def _fill_auxiliary_buses(net, ppc, bus_lookup, element, bus_column, aux):
     element_bus_idx = bus_lookup[net[element][bus_column].values]
     aux_idx = bus_lookup[aux[element]]
     ppc["bus"][aux_idx, BASE_KV] = ppc["bus"][element_bus_idx, BASE_KV]
-    if net._options["mode"] == "opf":
+    if hasattr(net, '_options'):
+        if net._options["mode"] == "opf" and element is "xward":
+            # the opf seems to have a tendency to optimize towards the lower bound of the internal voltage constraint.
+            # Also, if the delta value is smaller, the OPF does not converge. These observations may or may not apply
+            # to other net examples.
+            delta = 1e-1

```

Comment:
0.1 p.u. is a lot for voltage tolerance

Reply:
As mentioned in the comment above the value, the OPF did not converge with smaller delta values. That might be a problem with the example i tested with, though